### PR TITLE
feat(`monitoringp`): add consumer Consensus State in provider genesis

### PR DESCRIPTION
**Pull request by lubtd**
Add `chainID` (necessary to create the Client State) and Consensus State for consumer in the params of the module

During genesis initialization, these values are read to create the IBC client for the consumer. The Client ID is storing into the singleton `ConsumerClientID` and will be the only one allows to establish a connection with the Consumer chain

### CODE REVIEW

File: proto/monitoringp/query.proto [hunk 1]
```diff
@@ -15,7 +16,11 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/tendermint/spn/monitoringp/params";
   }
-  // this line is used by starport scaffolding # 2
+  // Queries a ConsumerClientID by index.
+	rpc ConsumerClientID(QueryGetConsumerClientIDRequest) returns (QueryGetConsumerClientIDResponse) {
```
Comment (Pantani) [inline proto/monitoringp/query.proto:20]: the indentation seems different here

File: x/monitoringp/client/cli/query.go [hunk 2]
```diff
@@ -25,6 +25,7 @@ func GetQueryCmd(queryRoute string) *cobra.Command {
 	}
 
 	cmd.AddCommand(CmdQueryParams())
+	cmd.AddCommand(CmdShowConsumerClientID())
```
Comment (Pantani) [inline x/monitoringp/client/cli/query.go:28]:
```suggestion
	cmd.AddCommand(
		CmdQueryParams(),
		CmdShowConsumerClientID(),
	)
```

File: x/monitoringp/client/cli/query_consumer_client_id.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/monitoringp/types"
+)
+
+func CmdShowConsumerClientID() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show-consumer-client-id",
+		Short: "shows consumerClientID",
```
Comment (Pantani) [inline x/monitoringp/client/cli/query_consumer_client_id.go:15]:
```suggestion
		Short: "shows the consumerClientID",
```

File: x/monitoringp/keeper/consumer_client_test.go [hunk 4]
```diff
@@ -0,0 +1,41 @@
+package keeper_test
+
+import (
+	"github.com/stretchr/testify/require"
+	testkeeper "github.com/tendermint/spn/testutil/keeper"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/monitoringp/types"
+	"testing"
```
Comment (Pantani) [inline x/monitoringp/keeper/consumer_client_test.go:8]:
```suggestion
	"testing"

	"github.com/stretchr/testify/require"
	testkeeper "github.com/tendermint/spn/testutil/keeper"
	"github.com/tendermint/spn/testutil/sample"
	"github.com/tendermint/spn/x/monitoringp/types"
```

File: x/monitoringp/types/errors.go [hunk 5]
```diff
@@ -8,7 +8,9 @@ import (
...
 // x/monitoringp module sentinel errors
 var (
-	ErrSample               = sdkerrors.Register(ModuleName, 1100, "sample error")
-	ErrInvalidPacketTimeout = sdkerrors.Register(ModuleName, 1500, "invalid packet timeout")
-	ErrInvalidVersion       = sdkerrors.Register(ModuleName, 1501, "invalid version")
+	ErrInvalidPacketTimeout  = sdkerrors.Register(ModuleName, 2, "invalid packet timeout")
+	ErrInvalidVersion        = sdkerrors.Register(ModuleName, 3, "invalid version")
+	ErrInvalidClientState    = sdkerrors.Register(ModuleName, 4, "invalid client state")
+	ErrInvalidConsensusState = sdkerrors.Register(ModuleName, 5, "invalid consensus state")
+	ErrClientCreationFailure = sdkerrors.Register(ModuleName, 6, "failed to create IBC client")
```
Comment (Pantani) [inline x/monitoringp/types/errors.go:15]:
```suggestion
	ErrClientCreationFailure = sdkerrors.Register(ModuleName, 6, "failed to create an IBC client")
```

File: x/monitoringp/types/genesis_test.go [hunk 6]
```diff
@@ -1,6 +1,7 @@
 package types_test
 
 import (
+	"github.com/tendermint/spn/testutil/sample"
 	"testing"
```
Comment (Pantani) [inline x/monitoringp/types/genesis_test.go:6]:
```suggestion
	"testing"

	"github.com/tendermint/spn/testutil/sample"
```

File: x/monitoringp/types/keys.go [hunk 7]
```diff
@@ -34,3 +34,7 @@ var (
 func KeyPrefix(p string) []byte {
 	return []byte(p)
 }
+
+const (
+	ConsumerClientIDKey = "ConsumerClientID-value-"
```
Comment (Pantani) [inline x/monitoringp/types/keys.go:39]: can we add comments to exported variables?

File: x/monitoringp/client/cli/query.go (hunk 2 above)
Comment (lubtd) [reply x/monitoringp/client/cli/query.go:28]: We should also put the params last

File: x/monitoringp/client/cli/query_consumer_client_id.go (hunk 3 above)
Comment (lubtd) [reply x/monitoringp/client/cli/query_consumer_client_id.go:15]: We can even put `shows the consumer client ID used for a chain IBC connection`

**Review by Pantani — APPROVED** [review]
CLI works well 👏🏼

**Pull request merged**